api/prometheus/v1/api_test.go

```diff
@@ -849,15 +867,14 @@ func TestAPIClientDo(t *testing.T) {
 				if err == nil {
 					t.Fatalf("expected error %q but got none", test.expectedErr)
```

Comment:
this can be simplified to:
`testutil.NotOk(t,err)`


also further down can use the testutil as well for:

```
if err != nil {
		t.Fatalf("unexpeceted error %s", err)
}
want, got := test.expectedBody, string(body)
if want != got {
	t.Errorf("unexpected body: want %q, got %q", want, got)
}
```